### ISSUE TITLE
Containerd periodically print logs that seem meaningless

### ISSUE THREAD

**Issue by haroldrandom** [fanux/sealos#644]
**那个命令或者组件**
containerd

```Shell
$ sealos version
Version: 3.3.9-rc.3
Last Commit: 4db495
```

kubernetes version: 1.20.0

**描述你的问题**
Containerd periodically (every 1 min almos) print logs that seem unnecessary:
```Shell
Jul  9 15:31:39 vm-k8s-server-1 systemd[1]: run-containerd-runc-k8s.io-f5e1123d3821ddcd5bbec244a163a594c2f771d6ec3a3e53ba6a254c72e56178-runc.S16L00.mount: Succeeded.
Jul  9 15:31:39 vm-k8s-server-1 systemd[804]: run-containerd-runc-k8s.io-f5e1123d3821ddcd5bbec244a163a594c2f771d6ec3a3e53ba6a254c72e56178-runc.S16L00.mount: Succeeded.
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.031064440Z" level=info msg="Finish piping \"stdout\" of container exec \"fa295ef46461979e46b255a6b4074c63d1f1cb82210ad3f5475937d1c901dea0\""
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.031439239Z" level=info msg="Finish piping \"stderr\" of container exec \"fa295ef46461979e46b255a6b4074c63d1f1cb82210ad3f5475937d1c901dea0\""
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.033818991Z" level=info msg="Exec process \"fa295ef46461979e46b255a6b4074c63d1f1cb82210ad3f5475937d1c901dea0\" exits with exit code 0 and error <nil>"
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.038953981Z" level=info msg="ExecSync for \"f5e1123d3821ddcd5bbec244a163a594c2f771d6ec3a3e53ba6a254c72e56178\" returns with exit code 0"
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.259932410Z" level=info msg="ExecSync for \"b4667e6274c1e8d2839510d90885b15071628e525111bedc85455d06d06a8f49\" with command [/usr/bin/check-status -r] and timeout 1 (s)"
Jul  9 15:31:40 vm-k8s-server-1 systemd[1]: run-containerd-runc-k8s.io-b4667e6274c1e8d2839510d90885b15071628e525111bedc85455d06d06a8f49-runc.OyNHUU.mount: Succeeded.
Jul  9 15:31:40 vm-k8s-server-1 systemd[804]: run-containerd-runc-k8s.io-b4667e6274c1e8d2839510d90885b15071628e525111bedc85455d06d06a8f49-runc.OyNHUU.mount: Succeeded.
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.359309116Z" level=info msg="Finish piping \"stdout\" of container exec \"16f9605770ab4cd1c3e246e727d83e894b2b1d3a12939c59f0f72a4a46160f28\""
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.359426956Z" level=info msg="Finish piping \"stderr\" of container exec \"16f9605770ab4cd1c3e246e727d83e894b2b1d3a12939c59f0f72a4a46160f28\""
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.361577001Z" level=info msg="Exec process \"16f9605770ab4cd1c3e246e727d83e894b2b1d3a12939c59f0f72a4a46160f28\" exits with exit code 0 and error <nil>"
Jul  9 15:31:40 vm-k8s-server-1 containerd[592]: time="2021-07-09T15:31:40.364106167Z" level=info msg="ExecSync for \"b4667e6274c1e8d2839510d90885b15071628e525111bedc85455d06d06a8f49\" returns with exit code 0"
```

what does this logs mean? 
If meaningless, can I reduce logs like this?

**你看到的一些参考资料**
eg. 文档的引用xxxx

**Comment by oldthreefeng** [comment]
there is an global flag for containerd , you can set it by your own.
`  --log-level value, -l value  set the logging level [debug, info, warn, error, fatal, panic]`

**Comment by haroldrandom** [comment]
> there is a global flag for containerd , you can set it on your own.
> ` --log-level value, -l value set the logging level [debug, info, warn, error, fatal, panic]`

Is there any convenient way to configure it globally? I have 1 master, 3 nodes, configuring them one by one is not a pleasant way :).

BTW, what's the default level for containerd to log that?

**Comment by oldthreefeng** [comment]
default level is `info`. 

>  not a pleasant way :).

pr will be welcome.

**Comment by haroldrandom** [comment]
I was wondering whether it's needed to upgrade containerd.

As this issue https://github.com/containerd/containerd/issues/4910 said, it's a bug to be resolved at 2020.01.08, and the corresponding version is containerd v1.5.0-beta.0. 

But upgrading is a huge risk task..... 

BTW, is there any open way to configure containerd on CLI or configuration files?
